lib/asciidoctor/syntax_highlighter/html_pipeline.rb

```diff
@@ -0,0 +1,10 @@
+module Asciidoctor
+  class SyntaxHighlighter::HtmlPipeline < SyntaxHighlighter::Base
+    register_for 'html-pipeline'
```

Comment:
I'm not familiar with this "highlighter", is it related to https://github.com/jch/html-pipeline ?

Reply:
It sets up the HTML so syntax highlighting will work on GitHub. (It's still a bit of a black box, but that's what it's designed for).

You can verify by making an AsciiDoc file in a GitHub repository that contains the following contents:

```
++++
<pre lang="ruby"><code>puts 'Hello, World!'</code></pre>
++++
```

GitHub only seems to consider the `lang` attribute on the `<pre>` tag. Technically, the `<code>` tag is optional, but I include it for semantic completeness.